Highlighted key number

xentenza opened the pull request:
- new Key number variant
- two versions: with or without link

xentenza committed:
Highlighted key number

- new Key number variant
- two versions: with or without link

williambelle commented on assets/components/organisms/key-number-group/key-number-group-hn.twig:
```diff
@@ -0,0 +1,6 @@
+<div class="container">
+  <div class="card-deck">
```
Je crois qu'il manque `</div>`  pour celui-ci.

williambelle commented on assets/components/molecules/key-number/key-number-hn-link.twig:
```diff
@@ -0,0 +1,23 @@
+{% set icon = icon|default("https://via.placeholder.com/") %}
```
Dans les autres fichiers `twig`, l'indentation est de 2.

xentenza committed:
corrections indentation + div manquant

- correction de l'indentation dans Molecules > Key Number

- <div> manquant dans Organisms > Key Number Group

williambelle committed:
Adjust
